states.go

```diff
@@ -13,7 +13,8 @@ const (
 	TaskState_TASK_LOST     TaskState = "TASK_LOST"
 	TaskState_TASK_ERROR    TaskState = "TASK_ERROR"
 	// Custom eremetic states
-	TaskState_TASK_QUEUED TaskState = "TASK_QUEUED"
+	TaskState_TASK_QUEUED      TaskState = "TASK_QUEUED"
+	TaskState_TASK_TERMINATING TaskState = "TASK_TERMINATING"
```

Comment:
Is there a reason for having two concepts for the same thing, e.g. _kill_ vs _terminate_? This signals to me that there might be other reasons for the task to terminate than manually killing it.

Will the task be marked as `finished` when successfully terminated? I think it would be more helpful to the developer if you could see which tasks that were forcibly stopped.